### PR TITLE
Fix PolylineCollection indices arrays (#5538)

### PR DESCRIPTION
**Pull request by fcayre**
Fix PolylineCollection indices arrays (#5538)

A 64K buffer limit overflow test was wrong, leading to out-of range vertice indices.

Current PolylineCollection tests are not broken by the change, which calls for a new or updated test.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@fcayre, thanks for the pull request!

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

__I am a bot who helps you make Cesium awesome!__ Thanks again.

**Comment by ggetz** [comment]
Hi @fcayre, thanks for the pull request!

Could you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) so we can review this?

**Comment by fcayre** [comment]
Hi @ggetz, you should have the document now, any update regarding the issue?

**Comment by pjcozzi** [comment]
@fcayre thanks again for the contribution!  I did not see your CLA, could you try emailing it directly to me: pcozzi@agi.com.  Thanks!

**Comment by pjcozzi** [comment]
We now have a CLA from @fcayre, thanks!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -152,4 +152,5 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 * [Rishabh Shah](https://github.com/rms13)
 * [Rudraksha Shah](https://github.com/Rudraksha20)
 * [Cody Guldner](https://github.com/burn123)
+* [Florent Cayré](https://github.com/fcayre)
```
Comment (pjcozzi) [inline CONTRIBUTORS.md]: Can you please add yourself under the corporate CLA section?

Comment (fcayre) [reply CONTRIBUTORS.md]: @pjcozzi done.

**Comment by ggetz** [comment]
Thanks @fcayre, looks good! Please update CHANGES.md with the bug you fixed and this is all good to go!

**Comment by fcayre** [comment]
@ggetz done, thanks.

**Comment by ggetz** [comment]
@pjcozzi I think this is good to merge

**Comment by mramato** [comment]
Verified the error is gone.  Thanks @fcayre!

**Pull request merged**